src/ChickenAPI/Packets/Game/Client/DropPacket.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace ChickenAPI.Packets.Game.Client
+{
+    [PacketHeader("drop")]
+    class DropPacket : PacketBase
+    {
+        [PacketIndex(0)]
+        public long ItemVNum { get; set; }
+
+        [PacketIndex(1)]
+        public short DropId { get; set }
```

Comment:
DropId should be a long (since it will be incremental, a short might not be enough before the EntityManager resets itself (and its counters so)